src/style.css

```
@@ -0,0 +1,90 @@
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: 100px;
+  font-size: 100px;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.box {
+  order: 0;
+  height: 300px;
+  color: #fff;
+  text-align: center;
+  line-height: 300px;
+  flex-grow: 1;
+}
+
+.first {
+  background-color: rgb(255, 0, 0);
+}
+
+.second {
+  background-color: rgb(204, 0, 0);
+}
+
+.third {
+  background-color: rgb(153, 0, 0);
+}
+
+.fourth {
+  background-color: rgb(102, 0, 0);
+}
+
+.fifth {
+  background-color: rgb(51, 0, 0);
+}
+
+.sixth {
+  background-color: rgb(0, 0, 0);
+}
+
+@media (max-width: 599px) {
+  .box {
+    flex-basis: 100%;
+  }
+}
+
+@media (min-width: 600px) and (max-width: 899px) {
```

```suggestion
@media (min-width: 600px) {
```
You may remove here `and (max-width: 899px)` because below you have another media query that will be applied on >=900px